docs/bootstrap_cces_aws.md

```diff
@@ -0,0 +1,48 @@
+## rubrik_bootstrap_cces_aws
+
+Bootstrap_cces_aws will complete the bootstrap process for a Rubrik Cloud Cluster ES on AWS. You will also need to configure the Rubrik provider with the "username" and "password" set to blank strings.
+
+## Example Usage
+
+```hcl
+resource "rubrik_bootstrap_cces_azure" "example" {
+  cluster_name           = "tf-demo"
+  admin_email            = "tf@demo.com"
+  admin_password         = "RubrikTFDemo2019"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- b7adca48-2bec-4479-90e8-1b6948a0aec0 -->